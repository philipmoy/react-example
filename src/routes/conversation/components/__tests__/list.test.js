import React from 'react';
import { shallow } from 'enzyme';

import List from '../list';

describe('ConversationList', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <List
        data={[]}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
