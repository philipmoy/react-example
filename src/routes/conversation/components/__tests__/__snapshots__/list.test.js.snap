// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversationList renders correctly 1`] = `
<div
  className="conversation-list"
>
  <div
    className="conversation-list__content"
  />
</div>
`;
